<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="connectivityGroupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>741</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Connecttivity</string>
    </property>
    <widget class="QPushButton" name="connectButton">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>40</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="brokerAddrTextbox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Broker address</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="brokerPortTextbox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Port</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>741</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Publish</string>
    </property>
    <widget class="QTextEdit" name="topicTextbox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Topic</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="messageTextbox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Message</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sendMessageButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>30</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
